draft-ietf-mops-streaming-opcons.md

```diff
@@ -98,7 +98,31 @@ informative:
     target: https://datatracker.ietf.org/doc/draft-iab-covid19-workshop/
     date: November 2020
 
+  codaspy:
+    title: Identifying HTTPS-Protected Netflix Videos in Real-Time
+    author:
+      - ins: Andrew Reed
+      - ins: Michael Kranch
+    target: http://library.usc.edu.ph/ACM/SIGSAC%202017/codaspy/p361.pdf
```

Comment:
> Should we ask the RFC editor about the best practices on this reference? I vaguely remember something about the most stable url or identifying description, and I'm not sure if it should be something like https://dl.acm.org/doi/10.1145/3029806.3029821 or the "CODASPY '17: Proceedings of the Seventh ACM on Conference on Data and Application Security and Privacy March 2017 Pages 361–368" text from there--I guess they'll probably flag this during the cleanup?

@GrumpyOldTroll - the reference format I'll have in a soon-to-be-submitted commit
```
  codaspy:
    title: "Identifying HTTPS-Protected Netflix Videos in Real-Time in CODASPY '17: Proceedings of the Seventh ACM on Conference on Data and Application Security and Privacy pp 361-368"
    author:
      - 
       initials: A.
       surname: Reed
       fullname: Andrew Reed 
      - 
       initials: M.
       surname: Kranch
       fullname: Michael Kranch
    target: https://dl.acm.org/doi/10.1145/3029806.3029821
    date: March 2017
```
gives 

```
   [codaspy]  Reed, A. and M. Kranch, "Identifying HTTPS-Protected
              Netflix Videos in Real-Time in CODASPY '17: Proceedings of
              the Seventh ACM on Conference on Data and Application
              Security and Privacy pp 361-368", March 2017,
              <https://dl.acm.org/doi/10.1145/3029806.3029821>.
```
I'm not happy about not being able to include multiple sets of quote marks, or whatever was blowing up my make, so the whole title is in quotes, but either we'll fix that or the production center will fix it.

Reply:
> Two levels of protection: media-level and transport-level. Media-level encryption is for business purposes, to avoid piracy, playback controls, etc. Transport-level encryption is for avoiding others from messing with your stuff (like manifest or media segments). As a note, transport-level encryption does not provide much privacy as looking at the payload even if it encrypted, third parties can figure out what is being streamed. I can provide some pointers to references that claimed to have achieved this.

@acbegen, I'm using some of your comment as text - see what you think when I do my next commit.